cypress/integration/pending-fragment.ts

```diff
@@ -105,4 +105,47 @@ context('Pending fragment', () => {
     cy.get('[data-cy=input]').type('@');
     cy.get('[data-cy=input]').textEqual('@ f');
   });
+
+  it('Should be able to autocomplete and reuse already typed "@"', () => {
+    cy.get('[data-cy=input]').type('a@{leftarrow}{backspace}{rightarrow}vi');
+
+    cy.get('[data-cy=input]').textEqual('@vi');
+    cy.get('[data-cy=pending]').should('have.text', '@vi');
+    cy.get('[data-cy=autocomplete]').should('exist', 1);
+  });
+
+  describe('Handle invalid spaces', () => {
+    it('Fixes problem with space before element', () => {
+      cy.get('[data-cy=input]').type(
+        'test{leftarrow}{leftarrow}{leftarrow}@{leftarrow}{backspace}{rightarrow} '
+      );
+
+      cy.get('[data-cy=input]').textEqual('@ est');
```

Comment:
I wonder why there is a space right after the `@`, in normal input it goes like this : `@est`.
```
test
t@est
@est
```

The case below is also puzzling to me with : ` @est` instead of `@est`
(The only change being the `{rightarrow}` at the end is not done.

Reply:
There is a `" "` after the `{rightarrow}`.
It's not really readable so I will move it to another `.type(' ')`

4706a1f